### PR TITLE
add <prometheus> in server settings and monitoring section

### PR DESCRIPTION
**Pull request by Slach**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
Signed-off-by: Slach <bloodjazman@gmail.com>

Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

**Comment by Slach** [comment]
related to https://github.com/ClickHouse/ClickHouse/pull/7900
https://github.com/prometheus/docs/pull/1513
@vdimir @alexey-milovidov  please look at this PR
@vdimir if this pull request will merge I think you can change the link in Prometheus docs

File: docs/en/operations/server_settings/settings.md [hunk 1]
```diff
@@ -531,6 +531,30 @@ The path to the directory containing data.
 <path>/var/lib/clickhouse/</path>
 ```
 
+## prometheus {#server_settings-prometheus}
+
+Expose metrics data for scraping from [Prometheus](https://prometheus.io).
```
Comment (BayoNet) [inline docs/en/operations/server_settings/settings.md:536]:
```suggestion
Exposing metrics data for scraping from [Prometheus](https://prometheus.io).
```

File: docs/en/operations/server_settings/settings.md [hunk 2]
```diff
@@ -531,6 +531,30 @@ The path to the directory containing data.
...
+
+Settings:
+
+-   endpoint – HTTP endpoint for scraping metrics by prometheus server. Begin from '/'.
```
Comment (BayoNet) [inline docs/en/operations/server_settings/settings.md:540]:
```suggestion
-   `endpoint` – HTTP endpoint for scraping metrics by prometheus server. Start from '/'. 
```

File: docs/en/operations/server_settings/settings.md [hunk 3]
```diff
@@ -531,6 +531,30 @@ The path to the directory containing data.
...
 ```
 
+## prometheus {#server_settings-prometheus}
+
+Expose metrics data for scraping from [Prometheus](https://prometheus.io).
+
+Settings:
+
+-   endpoint – HTTP endpoint for scraping metrics by prometheus server. Begin from '/'. 
+-   port – The listen port for prometheus HTTP endpoint.
```
Comment (BayoNet) [inline docs/en/operations/server_settings/settings.md:541]:
```suggestion
-   `port` – The listen port for `endpoint`.
```

File: docs/en/operations/server_settings/settings.md [hunk 4]
```diff
@@ -531,6 +531,30 @@ The path to the directory containing data.
...
+-   metrics – Expose current metrics values from the [system.metrics](../system_tables.md#system_tables-metrics) table.
+-   events – Expose current metrics values from the [system.events](../system_tables.md#system_tables-events) table.
```
Comment (BayoNet) [inline docs/en/operations/server_settings/settings.md:543]:
```suggestion
-   `events` – Flag that sets to expose metrics from the [system.events](../system_tables.md#system_tables-events) table.
```

File: docs/en/operations/server_settings/settings.md [hunk 5]
```diff
@@ -531,6 +531,30 @@ The path to the directory containing data.
...
+-   asynchronous\_metrics – Expose current metrics values from the [system.asynchronous\_metrics](../system_tables.md#system_tables-asynchronous_metrics) table.
```
Comment (BayoNet) [inline docs/en/operations/server_settings/settings.md:544]:
```suggestion
-   `asynchronous_metrics` – Flag that sets to expose current metrics values from the [system.asynchronous\_metrics](../system_tables.md#system_tables-asynchronous_metrics) table.
```

**Comment by BayoNet** [comment]
Don't forget to mirror changes to Russian version, please :-)

**Comment by Slach** [comment]
> Don't forget to mirror changes to Russian version, please :-)

done, I hope is enough 
@BayoNet please look at this PR again

**Review by BayoNet — APPROVED** [review]
It looks ok.

File: docs/ru/operations/server_settings/settings.md [hunk 6]
```diff
@@ -524,6 +524,30 @@ ClickHouse проверит условия `min_part_size` и `min_part_size_rat
 <path>/var/lib/clickhouse/</path>
 ```
 
+## prometheus {#server_settings-prometheus}
+
+Опубликовать данные о метриках, для сбора с помощью системы мониторинга [Prometheus](https://prometheus.io).
+
+Настройки:
+
+-   `endpoint` – путь по которому будет осуществляться экспорт данных метрик по HTTP протокоу для сбора с помощью prometheus. Должен начинаться с '/'.
```
Comment (qoega) [inline docs/ru/operations/server_settings/settings.md:533]:
```suggestion
-   `endpoint` – путь по которому будет осуществляться экспорт данных метрик по HTTP протоколу для сбора с помощью prometheus. Должен начинаться с '/'. 
```

**Review by qoega — APPROVED** [review]

**Comment by Slach** [comment]
@blinkov I fix conflicts with master could you merge this PR?

**Pull request merged**